pkg/cache/external/scheduler_cache.go

```diff
@@ -293,6 +293,21 @@ func (cache *SchedulerCache) ForgetPod(pod *v1.Pod) error {
 	return nil
 }
 
+func (cache *SchedulerCache) BindPodVolumes(pod *v1.Pod) error {
+	if cache.volumeBinder != nil {
+		if err := cache.volumeBinder.Binder.BindPodVolumes(pod); err != nil {
```

Comment:
We should skip calling BindPodVolumes when all volumes are bound already, otherwise the process will fail inside it. Perhaps we need to remember pods whose volumes need to be bound in cache after calling AssumePodVolumes(returns whether all volumes of this pod are bound).

Reply:
> We should skip calling BindPodVolumes when all volumes are bound already

Ideally, we should do that, which means when all PVCs are bound, we skip this call ... but our logic is different from the default scheduler, it is not easy to achieve this. 

> otherwise the process will fail inside it
I don't think it will fail. Let's say a pod uses 3 PVCs, 2 of them are bound and 1 is not., then it still needs to call BindPodVolumes, I don't expect it will fail, correct? In such case, when 3 PVCs are all bound, it should not fail too, it just might cause a little bit overhead.